player.h

```diff
@@ -130,15 +126,28 @@ void ensureUserPositionInLimits(int* userXPosition, int* userYPosition) {
 	}
 }
 
+Player playerFactory(){
```

Comment:
Factory isn't a POO pattern ?

Reply:
Indeed. I suggest rename the method. I believe that we can keep that modularized too.